packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx

```diff
@@ -245,7 +245,6 @@ const CompleteResetPassword = ({
             // NOTE: To fully sign in, totp must performed if 2FA is enabled.
             break;
           default:
```

Comment:
This default block has no effect. Remove it? Also was the TODO removed on purpose?